README.md

```diff
@@ -6,12 +6,12 @@
 A reference implementation of the [Cloud Native Buildpacks specification](https://github.com/buildpacks/spec).
 
 ## Supported APIs
-Lifecycle Version | Platform APIs              | Buildpack APIs |
-------------------|----------------------------|----------------|
-0.9.x             | [0.3][p/0.3], [0.4][p/0.4] | [0.2][b/0.2], [0.3][b/0.3], [0.4][b/0.4]
-0.8.x             | [0.3][p/0.3]               | [0.2][b/0.2]
-0.7.x             | [0.2][p/0.2]               | [0.2][b/0.2]
-0.6.x             | [0.2][p/0.2]               | [0.2][b/0.2]
+Lifecycle Version | Platform APIs                            | Buildpack APIs |
+------------------|------------------------------------------|----------------|
+0.9.x             | [0.3][p/0.3], [0.4][p/0.4], [0.5][b/0.5] | [0.2][b/0.2], [0.3][b/0.3], [0.4][b/0.4], [0.5][b/0.5]
```

Comment:
```suggestion
0.10.x            | [0.3][p/0.3], [0.4][p/0.4], [0.5][b/0.5] | [0.2][b/0.2], [0.3][b/0.3], [0.4][b/0.4], [0.5][b/0.5]
0.9.x             | [0.3][p/0.3], [0.4][p/0.4] | [0.2][b/0.2], [0.3][b/0.3], [0.4][b/0.4]
```

Reply:
Resolved in https://github.com/buildpacks/lifecycle/pull/509/commits/b0c266b366876f7d56730356c8a340b0b5254818